lib/unexpected-sinon.js

```diff
@@ -22,7 +22,7 @@
 
 (function (root, factory) {
     if (typeof exports === 'object') {
-        module.exports = factory(require('sinon'));
+        module.exports = factory(sinon || require('sinon'));
```

Comment:
This will most likely throw an error when sinon is not defined in the global scope. I recommend `root.sinon`

Also might make sense to revers the order, since the default behavior is preferred